hatchways/team-callback-cats
# create DB and connect mongo shell

### What this PR does (required):
- connect to mongoDB
- To start mongo shell, paste this code from server directory in terminal `mongosh "mongodb+srv://cluster0.mxbsd.mongodb.net/dogsitters" --username callbackCats      `    
- Password: j30nBRVSuHreiyF8

### Screenshots / Videos (required):
- ![Screen Shot 2021-08-09 at 8 33 08 PM](https://user-images.githubusercontent.com/63027785/128804285-fce97842-5a71-4cc8-bfbb-17e71ffb1f15.png)

### Any information needed to test this feature (required):
- Login to mongoAtlas with google account
      email callbackCats2021@gmail.com
      password Hatchways.?2021
- That will open our cluster. Within this our DB is called "dogsitters"
- Navigate to browse collection to see current database

- Mongo docs
  [(https://docs.mongodb.com/drivers/node/current/usage-examples/)]

### Any issues with the current functionality (optional):
- No issues so far I'm able to send data in from the shell
